modules/configuring-ibm-cos.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ibm-cloud.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-ibm-cos_{context}"]
+= Configuring {ibm-cloud-object-storage} instance
+
+You create an {ibm-cloud-object-storage} instance to store the {oadp-short} backup data. After you create the COS instance, you configure the COS instance with `HMAC` service credentials.
+
+.Prerequisites
+
+* You have an {ibm-cloud-title} Platform account.
+* You installed the link:https://cloud.ibm.com/docs/cli?topic=cli-getting-started[{ibm-cloud-title} Object Storage CLI].
```

Comment:
Users also required to install the cos plugin. 

`ibmcloud plugin install cos -f`

Reply:
Added this step. thanks.